### PR TITLE
chore: APPS-2712 Add GoogleTagsManager to Nuxt3

### PR DESCRIPTION
**Pull request by tinuola**
Connected to [APPS-2712](https://jira.library.ucla.edu/browse/APPS-2712)

- Add [Google Tags Manager package](https://nuxt.com/modules/nuxt-gtm) to Nuxt3 branch
- Add [GTM ID](https://github.com/UCLALibrary/library-website-nuxt/pull/285/files) to Nuxt config

**Checklist:**

-   [x] I added github label for semantic versioning
-   [x] I added notes
-   [x] I requested a PR review

[APPS-2712]: https://uclalibrary.atlassian.net/browse/APPS-2712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -101,6 +101,9 @@ export default defineNuxtConfig({
         || 'https://proxy.calendar.library.ucla.edu/',
       esTempIndexPrefixLibguides: process.env.ES_TEMP_INDEX_PREFIX_LIBGUIDES || '',
       esTempIndexLibguides: '',
+      gtm: {
+        id: 'GTM-T2SXV2'
```
Comment (farosFreed) [inline nuxt.config.ts:105]: Can we make this an .ENV variable? Or is this a public ID we don't need to hide?

Comment (tinuola) [reply nuxt.config.ts:105]: The tag was not hidden in Nuxt2 PR. And from my understanding of GTM, the ID has to be exposed on pages to be tracked.

Comment (farosFreed) [reply nuxt.config.ts:105]: Okay great, thanks for checking!

**Review by farosFreed — APPROVED** [review]
✅

**Pull request merged**